doc/sim65.sgml

```diff
@@ -104,6 +104,60 @@ PVExit ($01)
 </verb></tscreen>
 
 
+<sect>Creating a Test in C<p>
+
+For a C test compiled and linked with <tt/--target sim65/ the
+command line arguments to <tt/sim65/ will be passed to <tt/main/,
+and the return value from <tt/main/ will become sim65's exit code.
+The <tt/exit/ function may also be used to terminate with an exit code.
+ 
+Exit codes are limited to 8 bits.
+
+In addition to this, the simulator provides a set of built-in functions
+for simple file input and output:
+
+<tscreen><verb>
+  int open (const char* name, int flags, ...);
+  int __fastcall__ close (int fd);
+  int __fastcall__ read (int fd, void* buf, unsigned count);
+  int __fastcall__ write (int fd, const void* buf, unsigned count);
```

Comment:
It seems that it would be worth mentioning here that:
- STDIN_FILENO, STDOUT_FILENO and STDERR_FILENO are mapped to sim65's STDIN_FILENO, STDOUT_FILENO and STDERR_FILENO
- these functions allow to use the FILE*-based stdio as well (incl. stdin, stdout and stderr being mapped to sim65's stdin, stdout and stderr)

Reply:
Added.

Reply:
I think we have a misunderstanding here. I see two separate things (therefore the two bullets in my last comment):

1. open() and be used with STDIN_FILENO, STDOUT_FILENO and STDERR_FILENO (incl. short comment what it does).
2. fopen() can be used with stdin, stdout and stderr (incl. short comment what it does).

Your term "The standard C library file input and output" doesn't make it clear if you refer to 1.) or 2.). According to my understanding there's no single common terminology. I suggest something like "high-level FILE based functions" vs. "low-level file descriptor based functions".